schemas.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Validation schemas for incoming requests
+*/
+
+var Joi = require('joi');
+var mimes = Object.keys(require('mime-db'));
+
+exports.file = Joi.object().keys({
+  type: Joi.string().valid(mimes),
```

Comment:
Are there restrictions on which files we'll accept? Anyway, this is not going to be taken in the PUT/POST so the client doesn't really have control over this, the server determines what the mime type is.